src/client/runner/automation/playback/click/select-child-click.js

```diff
@@ -0,0 +1,199 @@
+import hammerhead from '../../../deps/hammerhead';
```

Comment:
we have some duplication in the file name: './_click_/select-child-_click_.js'. Maybe we can rename this somehow?